### DISCUSSION TITLE
Update Problem

### DISCUSSION THREAD

**Discussion by davidlattaro** [Koenkk/zigbee2mqtt#17912] [Devices]
after the last update ( 1.31.0-1), home assistant stopped recognizing the 4 gangs of this device:

https://www.zigbee2mqtt.io/devices/ZT-SR-EU4.html

**Comment by yarafie** [comment]
Same here.

**Comment by KoKolaj** [comment]
Same here.

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Comment by davidlattaro** [comment]
{"id":51,"type":"EndDevice","ieeeAddr":"0x70ac08fffe3f62bf","nwkAddr":33677,"manufId":4098,"manufName":"_TZ3000_a4xycprs","powerSource":"Battery","modelId":"TS0044","epList":[1,2,3,4],"endpoints":{"1":{"profId":260,"epId":1,"devId":0,"inClusterList":[0,1,6],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"65503":"��\u000f,\u0012","65506":31,"65508":0,"65534":0,"modelId":"TS0044","manufacturerName":"_TZ3000_a4xycprs","appVersion":66,"zclVersion":3,"powerSource":3}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":200,"batteryVoltage":30}}},"binds":[],"configuredReportings":[],"meta":{}},"2":{"profId":260,"epId":2,"devId":0,"inClusterList":[1,6],"outClusterList":[],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":260,"epId":3,"devId":0,"inClusterList":[1,6],"outClusterList":[],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}},"4":{"profId":260,"epId":4,"devId":0,"inClusterList":[1,6],"outClusterList":[],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{"configured":-708457359},"lastSeen":1686169930868,"defaultSendRequestWhen":"immediate"}

Em qua., 7 de jun. de 2023 às 16:53, Koen Kanters ***@***.***>
escreveu:

**Comment by Koenkk** [comment]
This issue should be fixed in z2m 1.31.2

**Reply by davidlattaro** [reply]
thank you
